src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -225,15 +228,16 @@ private void VerifyAdditionalFile(AdditionalText file)
         /// <summary>
         /// Returns diagnostics produced by all <see cref="Analyzers"/>.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public Task<ImmutableArray<Diagnostic>> GetAnalyzerDiagnosticsAsync()
```

Comment:
can't warn on this, as we often have APIs where user provided cancellationtokens are optional, and we want to allow users to call those without a CT for simple programming purposes.  We do hide this though, so that the user only sees the method below, which follows our standard api pattern of an actually 'optional' parameter.